tools/js-sdk-release-tools/src/llc/generateRLCInPipeline/generateRLCInPipeline.ts

```diff
@@ -77,7 +77,9 @@ export async function generateRLCInPipeline(options: {
                 logger.info(`Start to run command: '${scriptCommand}'`);
                 execSync(scriptCommand, {stdio: 'inherit'});
                 logger.info("Generated code by tsp-client successfully.");
-            } 
+            }
+            packagePath = generatedPackageDir;
```

Comment:
i didn't see an error from your ci link: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4578488&view=logs&j=47092895-c7c8-567a-c7a6-8c88f3ec178e&t=d293a8e8-c6af-5091-cdf7-4c66482dd6f2&l=6927

Reply:
it's fixed, so there's no error

Reply:
The failure is here: [Pipelines - Run Azure OpenAPI logs](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4578096&view=logs&j=47092895-c7c8-567a-c7a6-8c88f3ec178e&t=d293a8e8-c6af-5091-cdf7-4c66482dd6f2)